eslint.config.mjs

```diff
@@ -419,21 +420,32 @@ export default tseslint.config([
       'turbo/no-undeclared-env-vars': 'off',
     },
   },
-  ...pluginYml.configs['flat/recommended'],
   {
-    name: 'repo/.github',
-    // rules: {
-    // 'regex/invalid': [
-    //   'error',
-    //   [
-    //     {
-    //       regex: '^(?!.*\\$TURBO_ARGS( |$)).*turbo \\S+',
-    //       message: 'Invalid turbo CI command. Must contain `$TURBO_ARGS`',
-    //     },
-    //   ],
-    // ],
-    // },
+    name: 'repo/jsdoc',
+    ...pluginJsDoc.configs['flat/recommended-typescript'],
+    files: ['packages/shared/src/**/*.{ts,tsx}'],
+    ignores: ['**/__tests__/**'],
+    plugins: {
+      jsdoc: pluginJsDoc,
+    },
+    rules: {
+      ...pluginJsDoc.configs['flat/recommended-typescript'].rules,
+      'jsdoc/check-examples': 'off',
```

Comment:
I don't want it to raise false positives in the examples, we want more control over it